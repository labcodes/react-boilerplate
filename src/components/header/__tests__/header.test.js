'use strict';

import React from 'react';

// Test helpers
import { shallow, mount, render } from 'enzyme';
import renderer from 'react-test-renderer';

// Component
import Header from 'components/header';

// ====

describe('Header [Snapshot]', () => {
    it('Renders correctly', () => {
        const tree = renderer.create(<Header />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});

