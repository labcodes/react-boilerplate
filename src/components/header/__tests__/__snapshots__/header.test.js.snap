// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header [Snapshot] Renders correctly 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
    src=""
  />
  <h1
    className="App-title"
  >
    Welcome to React
  </h1>
  <button
    onClick={[Function]}
  >
    change user status
  </button>
</header>
`;
